tm2py/components/network/skims.py

```diff
@@ -80,24 +82,65 @@ def get_omx_skim_as_numpy(
     _trn_classes = {c.name: c for c in controller.config.transit.classes}
     _nm_classes = {c.name: c for c in controller.config.active_modes.classes}
 
-    if skim_mode in _hwy_classes.keys():
-        _config = controller.config.highway
-        _mode_config = _hwy_classes[skim_mode]
-    elif skim_mode in _trn_classes.keys():
-        _config = controller.config.transit
-        _mode_config = _trn_classes[skim_mode]
-    elif skim_mode in _nm_classes.keys():
-        _config = controller.config.active_modes
-        _mode_config = _nm_classes[skim_mode]
 
+    if force_read:
+        if force_read[0] == 'highway':
```

Comment:
I'm not convinced that we need to add `force_read`. Why not change the class specification in the accessibility configs and feed that in? Like instead of `da`, the user can define the class name that matches the ones in skims (could be da, daxh, dah, dam, dal)
https://github.com/camsys/tm2py/blob/b88c4666c181634928740be15c69ec7390d6158b/examples/model_config.toml#L102

Reply:
Reversed.

Reply:
Have you tested the updated settings with `dah`?